lib/msal-core/src/ScopeSet.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export class ScopeSet {
```

Comment:
Better name? May be 'ScopesUtils'?

Reply:
This file will be an object that represents the set of scopes passed to MSAL by the developer. Ideally it will perform all scope related functions, and act upon its own object, so we won't need a ScopesUtils.